.eslintrc.json

```diff
@@ -1,3 +1,7 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": "next/core-web-vitals",
+  "rules": {
+    "react/no-unescaped-entities": "off",
+    "@next/next/no-page-custom-font": "off"
```

Comment:
To use the full advantages of es-lint do not turn off rules and push them to github. This is a bad practice. You're basically defeating the purpose/use of eslint in general.